package.json

```diff
@@ -19,7 +19,7 @@
     "url": "https://github.com/gtramontina/ghooks/issues"
   },
   "scripts": {
-    "lint": "eslint bin/ lib/ test/",
+    "lint": "eslint bin/ lib/* test/",
```

Comment:
Hmm... had a quick look, and yes, `./bin/install` does not get linted, too (does not comply to lint rules either).